e2e/commands_test.go

```diff
@@ -47,6 +47,15 @@ func TestRenderTemplates(t *testing.T) {
 	}
 }
 
+func TestRenderAttachment(t *testing.T) {
+	skip.If(t, !hasExperimental, "experimental mode needed for this test")
+	result := icmd.RunCommand(dockerApp, "render-attachment",
+		"testdata/helm.dockerapp",
+		"testdata/attachment.yml",
```

Comment:
I don't understand how does that work 🤔 Shouldn't the attachement be nested **inside** the app package? And the file path, I think it should be relative to the application package.

So here we should have `attachment.yml` inside `testdata/helm.dockerapp` directory, and the file path should be just `attachment.yml`.

Reply:
Because if it's not in the dockerapp directory then it's not an attachment.

Reply:
Just to clarify things here. The following is I think the main usage of this feature, ie. from a remote image.

```
docker-app render-attachment garethr/hello attachment.yml 
```

In the context of a directory I'd expect that to be:

```
docker-app render-attachment path/to/hello attachment.yml 
```

If you're in the app directory I'd expect that to be:

```
docker-app render-attachment attachment.yml 
```

I think it's only the later where shell completion is relevant I feel.